docs/users.md

```diff
@@ -314,6 +316,12 @@ To create a control plane of type `host` run the command:
 kflex create cp4 --type host
 ```
 
```

Comment:
Users of the API are going to need an adequate explanation too. The documentation below for CLI users omits all mention of the bootstrap secret.